src/content/app-platform/create-catalog/index.md

```diff
@@ -0,0 +1,164 @@
+be served over HTTP.
+
+We recommend using our [app-build-suite](https://github.com/giantswarm/app-build-suite/)
+tool to publish apps to your catalog. It adds additional metadata files that
+allows app platform to extend Helm. Such as only allowing an app to be
+installed once in a cluster. You can learn more about the tool by reading its
+[tutorial](https://github.com/giantswarm/app-build-suite/blob/master/docs/tutorial.md).
+
+These metadata files and the Helm `index.yaml` are used to generate app metadata
+that is stored in the management cluster in [App Catalog Entry]({{< relref "/ui-api/management-api/crd/appcatalogentries.application.giantswarm.io.md" >}})
+CRs.
+
+There are [multiple options](https://helm.sh/docs/topics/chart_repository/#hosting-chart-repositories)
+for serving the catalog over HTTP including GitHub Pages or tools like Harbor or ChartMuseum which run in a
+Kubernetes cluster. At Giant Swarm we use GitHub Pages and this is what we will
+cover in this guide.
+
+## Creating an app catalog hosted using GitHub Pages
+
+First, you should choose a name for your catalog. We recommend using the suffix
+`-catalog` to make it clear this Git repository hosts an app catalog.
+
+Create the Git repository in GitHub as a public repository and enable
+[GitHub Pages](https://docs.github.com/en/pages/quickstart) for the `main` branch.
+
+## Publishing an app to the app catalog
+
+Now you can configure your apps to be published to your catalog. We have created
+a GitHub Action for `app-build-suite` that will automate this.
+
+```yaml
+# .github/workflows/push-to-app-catalog.yaml
+name: 'Push to App Catalog'
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  push_to_app_catalog:
+    uses: giantswarm/app-build-suite/.github/workflows/push-to-app-catalog.yaml@v1.1.0
+    with:
+      app_catalog: example-catalog
+      chart: hello-world-app
+      organization: example-org
+    secrets:
+      envPAT: ${{ secrets.PAT }}
+```
+
+To configure the action you need to add a secret called `PAT` to the app's git
+repository. This must contain a [Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+that has write permission to the app catalog git repository.
+
+We recommend using [Dependabot](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/keeping-your-actions-up-to-date-with-dependabot)
+to keep the GitHub Action up to date.
+
+## Create Catalog CR
+
+Once you've created your app catalog and published some apps you need to create
+a Catalog CR in the management cluster to register it with app platform. This
+can be done using [kubectl gs template catalog]({{< relref "/ui-api/kubectl-gs/template-catalog" >}}).
+
+You should create the Catalog CR in the [organization]({{< relref "/general/organizations" >}})
+namespace where the apps will be used.
```

Comment:
Any alternative, if the catalog is supposed to be used by more than one org? Use the default namespace?

Reply:
Yes, default would be the alternative if the catalog is needed for multiple orgs and customers should have permission. I'll update this section.